smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/bindingTraits/HttpResponseTraitWithoutHttpPayload.kt

```diff
@@ -49,16 +49,19 @@ class HttpResponseTraitWithoutHttpPayload(
         val bodyMembersWithoutQueryTrait = bodyMembers
             .filter { !it.member.hasTrait(HttpQueryTrait::class.java) }
             .toMutableSet()
+        val initialResponseMembers = bodyMembers.filter {
+            val targetShape = it.member.targetOrSelf(ctx.model)
+            targetShape?.hasTrait(StreamingTrait::class.java) == false
+        }.toMutableSet()
```

Comment:
nit: Maybe use `toSet()` instead of `toMutableSet()` if it's not gonna be modified / shouldn't be modified except at this point in code?